terragrunt/modules/release-distribution/lambdas/doc-router/index.js

```diff
@@ -66,10 +66,17 @@ exports.handler = (event, context, callback) => {
         return temp_redirect('https://www.rust-lang.org/learn', callback);
     }
 
-    // Forward versioned documentation as-is.
-    if (/^\/\d/.test(request.uri)) {
+    // Forward patch versioned documentation as-is.
+    if (/^(\d+)\.(\d+).(\d+)\/(.*)/.test(request.uri)) {
```

Comment:
This regex looks good 👍
![image](https://github.com/user-attachments/assets/dc2a9b9d-2a3a-45f2-8df0-76d37e01127d)